types/react-dom/experimental.d.ts

```diff
@@ -50,4 +52,25 @@ declare module '.' {
     type FormStatus = FormStatusPending | FormStatusNotPending;
 
     function experimental_useFormStatus(): FormStatus;
+
+    function experimental_useFormState<State>(
+        action: (state: State) => Promise<State>,
```

Comment:
@eps1lon Why is the args type the same as the returned one?
In the next docs, we should be able to accept a FormData, then return our own custom state. Am I missing something?